language/ftn08/api.tex

```diff
@@ -0,0 +1,365 @@
+As the \Fortran[bind(C)] language bindings are implemented over the \openshmem
```

Comment:
Please move all of these files from directory "ftn08" to directory "fortran2008" to more closely match the referred-to named "ISO/IEC 1539-1:2010 (Fortran 2008)". Or "fortran2018", or just "fortran" if you never anticipate ever having two separate sets of FortranXXXX language bindings.

Reply:
Nick had a comment that if these bindings are for Fortran 2018 because of its support for AMOs, collectives, etc., this document should make that comment somewhere so that users know why the C_BIND interface for AMOs, collectives are missing.